generators/app/utils.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const R = require('ramda');
+
+/**
+ * Turn a string into a nice title.
+ * @param {String}
+ * @returns {String}
+ * @example toTitle('some-slug_string'); => Some Slug String
+ */
+const toTitle = R.pipe(
+  // String => Array
+  R.split(/[\W_]+/),
+  // Array => Array
+  R.map(word => word.replace(/^\w/, R.toUpper)),
+  // Array => String
+  R.join(' ')
+);
+
+/**
+ * Turn a string into a slug.
+ * Inspired by https://gist.github.com/juanmhidalgo/3146760
+ * @param {String}
+ * @returns {String}
+ * @example toSlug('Nice Title!'); => nice-title
+ */
+const toSlug = R.pipe(
```

Comment:
I actually installed `slug` at first. I didn't check the actual size, but got scared away by how big it _seemed_ during the install (it comes with a lot of language support). Silly, I know. I'll check the real size and reconsider.